tests/data/remove_for_brackets.py

```diff
@@ -0,0 +1,18 @@
+# Only remove tuple brackets after `for`
+for (k, v) in d.items():
```

Comment:
Can you add a test where the for loop target is too long to fit in one line?

Reply:
Thanks, added now 😄